Model/MediaPlatform.php

```diff
@@ -25,6 +25,7 @@ class MediaPlatform extends AuthManagerAppModel {
 	const INSTAGRAM = 11;
 	const DROPBOX = 12;
 	const ADWORDS = 13;
+	const LINKED_IN = 14;
```

Comment:
Since we already have `LINKED_IN_ADS`, can we name this one eg. `LINKED_IN_PAGES`